install/Knative-custom-install.md

```diff
@@ -25,11 +25,11 @@ that you to run multiple and separate installation commands.
 
 - Kubernetes requirements:
 
-  - Your Kubernetes cluster version must be v1.10 or newer.
+  - Your Kubernetes cluster version must be v1.11 or newer.
 
   - Your version of the
     [`kubectl` CLI tool](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-    must be v1.10 or newer.
+    must be v1.11 or newer.
```

Comment:
Through testing, it's been determined that kubectl 1.10 is still fine, so leave the requirement for kubectl at 1.10 throughout this PR.